products/cakephp.md

```diff
@@ -19,222 +18,49 @@ auto:
 -   git: https://github.com/cakephp/cakephp.git
 
 releases:
--   releaseCycle: "4.4"
```

Comment:
As I mentioned this [before](https://github.com/endoflife-date/endoflife.date/pull/1707#issuecomment-1282136426), minor releases do hold significance for the CakePHP project.

For example, a very recent security patch was released for 4.2, 4.3 & 4.4.

https://bakery.cakephp.org/2023/01/06/cakephp_4211_4311_4410_released.html

If we delete all the 4.x minors and only show 4.x as one big release we remove a lot of info for people who still use older but obviously still supported 4.x minor releases.

Reply:
Got it, I will update the product description with that information.

It seems usually a patch is issued for the last three versions (https://bakery.cakephp.org/2023/01/06/cakephp_4211_4311_4410_released.html, https://bakery.cakephp.org/2019/04/23/cakephp_377_3615_3518_released.html, https://bakery.cakephp.org/2018/05/20/cakephp_364_3517_3414_released.html). Can I use that rule or should I be more vague ?

Reply:
Closing this in favor of #2291.